py-glaredb/justfile

```diff
@@ -2,15 +2,14 @@ set fallback
 VENV := ".venv"
 VENV_BIN := ".venv/bin"
 
-
 @venv:
-	@python3 -m pip install virtualenv > /dev/null
-	@python3 -m virtualenv {{VENV}} > /dev/null
+	python3 -c "import virtualenv" || python3 -m pip --quiet install virtualenv
+	python3 -m virtualenv {{VENV}} --quiet
 
 ## Set up virtual environment and install requirements
-@requirements: venv
-	{{VENV_BIN}}/python -m pip install --upgrade pip > /dev/null
-	{{VENV_BIN}}/pip install -r requirements.txt > /dev/null
+@requirements: {{VENV}}
```

Comment:
shouldn't this still be `venv` specifying that the other command `@venv` is a dependency of this one?

I'm actually surprised `just` doesn't complain with it like this as there is no command `.venv`